### ISSUE TITLE
Implement Hydra node requests on delegate server

### ISSUE THREAD

**Issue by uhbif19** [mlabs-haskell/hydra-auction#158]
Most of things should be already covered by #124 and #125 

Domain logic should be covered by other tasks.

Scope of this task is to actually react on `DelegateEvents` by triggering actions on L2. Concretelly:

- [ ] Introduce Hydra node and Cardano node connection in config or CLI option
- [x] Init/Abort/Close Head
- [x] Commit - both empty and StandingBid transactions case
- [x] Placing new bid
- [x] Probably testing this in E2E test?

No signing by bidder is required in this milestone.

Blocked by #140 
Blocked by #124 
Blocked by #125
